header-verifier/tests/header_verifier_blackbox_setup.rs

```diff
@@ -213,8 +213,18 @@ impl HeaderVerifierTestState {
         self.common_setup
             .assert_expected_error_message(response, None);
 
+        let expected_logs = if let Some(execution_error) = execution_error {
+            Some(vec![
+                log!(CHANGE_VALIDATOR_SET_ENDPOINT, topics: [EXECUTED_BRIDGE_OP_EVENT], data: execution_error),
+            ])
+        } else {
+            Some(vec![
+                log!(CHANGE_VALIDATOR_SET_ENDPOINT, topics: [EXECUTED_BRIDGE_OP_EVENT]),
+            ])
+        };
+
         self.common_setup
-            .assert_expected_log(logs, expected_custom_log, expected_log_error);
+            .assert_expected_log_refactored(logs, expected_logs);
```

Comment:
I think you can just use this for `expected_logs` because if execution_error is none it's fine
```
Some(vec![log!(CHANGE_VALIDATOR_SET_ENDPOINT, topics: [EXECUTED_BRIDGE_OP_EVENT], data: execution_error)])
```